index.js

```diff
@@ -80,6 +80,28 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = async input => {
+	let i = 0;
+
+	return new Promise(resolve => {
+		const checker = setInterval(async () => {
+			i++;
+			const exists = await processExists(input);
+
+			if (!exists) {
+				resolve();
+				clearInterval(checker);
+			}
+
+			// Give up after 2 seconds
```

Comment:
Why 2 seconds? And the 2 seconds should be a constant, so it can easily be changed if needed.